src/interfaces/pyo3/tests/test.sh

```diff
@@ -2,15 +2,13 @@
 
 set -euEx
 
-init(){
-  virtualenv env
-  source env/bin/activate
-  pip install maturin
-}
+# Clean previous build
```

Comment:
should we finally keep this `test.sh` file? not used anywhere else?

Reply:
I kept the file to quickly build and test the python package on one's local pc.